add fields to support servers-side node size (#221)
opencadc/vos: cavern/README.md

# (optional) keys to generate pre-auth URLs to cavern: now generated internally

# (optional) base directory exposed for sshfs mounts
org.opencadc.cavern.sshfs.serverBase = {server}[:{port}]:{path}
```

The _resourceID_ is the resourceID of _this_ `cavern` service.

The _allocationParent_ is a path to a container node (directory) which contains space allocations. An allocation
is owned by a user (uisually different from the _rootOwner_ admin user) who is responsible for the allocation
and all conntent therein. The owner of an allocation is granted additional permissions within their 
allocation (they can read/write/delete anything) so the owner cannot be blocked from access to any content
within their allocation. This probably only matters for multi-user projects. Multiple _allocationParent_(s) may
be configured to organise the top level of the content (e.g. /home and /projects). Paths configured to be 
_allocationParent_(s) will be automatically created (if necessary), owned by the _rootOwner_, and will be

> anonymously readable (public). Limitation: only a single level of top-level _allocationParent_(s) are supported.

Reviewer: ```suggestion
anonymously readable (public). Limitation: only top-level container nodes can be configured as _allocationParent_(s).
```
